windows/libdatadog.csproj

```diff
@@ -39,27 +39,35 @@
       PackagePath="include\native\datadog\data-pipeline.h" />
 
     <None Include="$(LibDatadogBinariesOutputDir)\x86_64-pc-windows-msvc\debug\datadog_profiling_ffi.lib"
+      Pack="true" PackagePath="build\native\lib\x64\debug\static\datadog_profiling_ffi.lib" />
+    <None Include="$(LibDatadogBinariesOutputDir)\x86_64-pc-windows-msvc\debug\datadog_profiling_ffi.dll.lib"
```

Comment:
/question

what's the intention behind the extension change? Name collision?

Reply:
The `dll.lib` is produced by cargo.
Since the static lib `*.lib` is produced in the same folder as the dynamic ones `*.lib + *.dll`, so yes name collision avoiding.